services/language_translation/v2.js

```diff
@@ -152,29 +108,96 @@ module.exports = function (RED) {
         version: 'v2'
       });
 
-      var model_id = "";
-      if(domain === "news") {
+      switch (action) {
+        case 'translate':
+          this.doTranslate(msg);
+          break;
+        case 'train':
+          this.doTrain(msg);
+          break;
+        case 'getstatus':
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2582431672/issues/source?bid=3225115&fileBranchId=3303298#l118)